resources/js/Pages/Users.tsx

```diff
@@ -212,61 +278,93 @@ export default function Users({ auth }: PageProps) {
             <dialog ref={resetUserPasswordModal} className="modal">
                 <div className="modal-box">
                     <h3 className="font-bold text-xl">
-                        Reset User's Password?
+                        <p>Reset User's Password?</p>
                     </h3>
+                    <h4 className="font-bold text-l text-secondary">
+                        <br />
+                        <p>Note: Only for non-administrator accounts.</p>
+                    </h4>
                     <p className="py-4"></p>
                     <div className="modal-action">
-                        <button className="btn btn-warning">Reset</button>
+                        <button
+                            className="btn btn-warning"
```

Comment:
For consistency sake (until we get a component that is used for all modals), could you modify these modals to follow the same format as left menu management modals? ie. color should be btn-error (or you can change the left menu modals to btn-warning... either way), close button should be left aligned, and all modals should have an "x" on the top right corner. Super small changes (and I know they existed before your PR) but keeps the UI consistent!